parameters/examples/genesis.rs

```diff
@@ -0,0 +1,200 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
+// The snarkVM library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkVM library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
+
+use snarkvm_algorithms::CRH;
+use snarkvm_dpc::{prelude::*, testnet1::*, testnet2::*};
+use snarkvm_ledger::{
+    posw::{txids_to_roots, PoswMarlin},
+    prelude::*,
+};
+use snarkvm_utilities::{to_bytes_le, ToBytes};
+
+use rand::{thread_rng, Rng};
+use std::{
+    fs::File,
+    io::{Result as IoResult, Write},
+    path::Path,
+    str::FromStr,
+};
+
+pub fn generate<C: Parameters>(recipient: &Address<C>, value: u64) -> Result<(Vec<u8>, Vec<u8>), DPCError> {
+    let rng = &mut thread_rng();
+
+    // TODO (howardwu): Deprecate this in favor of a simple struct with 2 Merkle trees.
+    let temporary_ledger = {
+        let mut temporary_path = std::env::temp_dir();
+        temporary_path.push(format!("./test_db-{}", rand::thread_rng().gen::<usize>()));
+
+        Ledger::<C, MemDb>::new(Some(&temporary_path), Block {
```

Comment:
the path should not be needed if `MemDb` is used, i.e. this call should rather be:
```
Ledger::<C, MemDb>::new(None, Block {
```

Reply:
Good catch, will fix